### DISCUSSION TITLE
Affect how data from another collection is displayed

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#12893] [Feature Requests]
Now as you see the data from the second collection in the range of the column item is displayed with "," without spaces or something. It would be good to be able to change it to make it more readable and eg apply " - "
It would be great if the top bar was still visible, it is better to read the data when you move the list down...

![Zrzut ekranu 2022-04-20 204305](https://user-images.githubusercontent.com/90528895/164306537-b50f5961-2d4d-4bc9-be94-676c9278ed93.jpg)

At the same time, I installed the add-on [directus-extension-display-link](https://github.com/jacoborus/directus-extension-display-link) and it causes that only after refreshing the data is visible - I know the fault of the add-on. Just for your information. On this example, it may be better if you could somehow combine data. The same data is repeatedly combined with each other. If there is a single data still correctly leads to the link. What if the marketplace when have such add-ons and will also do it...

https://user-images.githubusercontent.com/90528895/164303489-8281ed5c-152f-410e-bfa4-3dba6d3bb856.mp4

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/issues/12271 🙂

**Reply by koksikus** [reply]
Sorry, I did not treat it as a bug so I did not even look there ;) 
"Display link" cool thing just a pity that it is developed outside the project and causes an error.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
